README.md

```diff
@@ -0,0 +1,84 @@
+# Ada Pets Node
+
+<!-- Note: this was based on Ada Pets React no the other way around. -->
+
+## At a Glance
+
+- Individual, [stage 1](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md#stage-1) project.
+- To be completed in class on **DATE**. No pull request is required.
+
+## Learning Goals
+
+* Reading and understanding existing JavaScript code
+* Using axios to read data from an API in JavaScript
+* Parsing through an API response in JavaScript and extracting desired information
+* Using axios to write data to an API in JavaScript
+
+## Introduction
+
+Ada Lovelace is starting a pet rescue service advertising pets in need of rescue via an API.   This app should load the list of pets automatically and present a menu with options.   The user should be able to list all pets, select a pet and see details, add new pets and remove a pet from the App once adopted. 
+
+### Setup
+
+1. Fork and clone this repository
+2. Run `npm install` to install dependencies
+3. Run the app with `node src/main.js`
+
+### Layout
+
+You will be adding your code to `src/main.js`.  You should read through this file to start to get an idea of the basics of how it works.  All of the functions you need to implement are currently there, just with empty bodies.
+
+There are tests for each of the waves in the `test/` directory, there is one file per wave.
```

Comment:
I'd love a section that introduces the files that will already be there, namely just high-level explanation of the axios mocking that will happen